segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/SystemJournal.java

```diff
@@ -140,7 +150,7 @@ public SystemJournal(int containerId, long epoch, ChunkStorage chunkStorage, Chu
         this.systemSegments = getChunkStorageSystemSegments(containerId);
         this.systemSegmentsPrefix = NameUtils.INTERNAL_SCOPE_NAME;
 
-        Preconditions.checkState(!chunkStorage.exists(getSystemJournalChunkName()));
+        Preconditions.checkState(!chunkStorage.exists(getSystemJournalChunkName()).get());
```

Comment:
Is there a plan to fix these?

Reply:
not yet. This happens at the very beginning when these are the only SLTS operations at the bootstrap time. There is no table store access here.